cla_public/apps/checker/session.py

```diff
@@ -145,6 +145,20 @@ def aged_60_or_over(self):
     def callback_time(self):
         return self.get('CallMeBackForm_time')
 
+    @property
+    def eligibility_status(self):
+        status = 'unknown'
+        is_eligible = self.get('is_eligible', ELIGIBILITY_STATES.UNKNOWN)
```

Comment:
Currently `is_eligible` is set only after completing “About you” form. Therefore `eligibility_status` will throw when it's accessed from pages before to completing the “About you” form (i.e. going straight to “Call me back” page).